.eslintignore

```diff
@@ -1,18 +1,19 @@
 /.git
-/docs/.next
 /coverage
+/docs/.next
 /docs/export
 /examples/create-react-app*/src/serviceWorker.js
 /examples/gatsby/public/
 /examples/preact/config
 /examples/preact/scripts
+/framer/Material-UI.framerfx/design/document.json
 /packages/babel-plugin-unwrap-createStyles/test/__fixtures__/
 /packages/material-ui-codemod/lib
 /packages/material-ui-codemod/src/*/*.test
 /packages/material-ui-codemod/src/*/*.test.js
 /packages/material-ui-icons/fixtures
-/packages/material-ui-icons/src
 /packages/material-ui-icons/legacy
+/packages/material-ui-icons/src
```

Comment:
I don't think we want to not lint these. We should make the script understand an additional prettierignore that extends eslintignore

Reply:
What's the use case for linting these 6,000 generated icons? (to be clear, I have sorted ASC the file, the ignore was already in place)

Reply:
> What's the use case for linting these 6,000 generated icons?

that the generated code is sound

> I have sorted ASC the file, the ignore was already in place

I want to repeat that sorting ASC should not be done in substantial PRs. It makes way more sense to group these ignores by purpose and by randomly changing the sort you disrupt git history and reviews.

If this wasn't changed then we can keep it.